### ISSUE TITLE
Updates to Accessibility "Research and Design" Page

### ISSUE THREAD

**Issue by emily-phet** [phetsims/website#437]
Updates/corrections/additions to Publication List:

- New Reference for top of "Accessible Interactive PhET Simulations" list: 
Taliesin L. Smith. Access, action, & agency: Inclusive design for the non-visual use of a highly interactive science simulation. Masters in Inclusive Design, Major Research Project. OCAD University, Toronto, Canada. In Press.

- Existing Reference: 
Emily Randall. Making science simulations accessible for students with vision impairments. Proceedings of the ACM SIGCHI Conference on Human Factors in Computing Systems (CHI). In Press.
Should be updated: 
Emily Randall. Making science simulations accessible for students with vision impairments. (2016) _Proceedings of the 34th Annual ACM Conference Extended Abstracts on Human Factors in Computing Systems_ (pp. 122-127). ACM. Paper. [link: 10.1145/2851581.2890385]

- Existing Reference: 
Emily Randall. Developing accessible physics simulations for students with vision impairments. Physics Education and Computer Science Honors Thesis. Defended March 18, 2016.
Should read:
Emily Randall. Developing accessible physics simulations for students with vision impairments. (2016) Physics Education and Computer Science Undergraduate Honors Thesis. University of Colorado Boulder, Boulder, CO. In Press. [no link - current link redirects to phet site]

- Existing Reference: 
Emily B. Moore. ConfChem Conference on Interactive Visualizations for Chemistry Teaching and Learning: Accessibility for PhET Interactive Simulations—Progress, Challenges, and Potential. Journal of Chemical Education. In Press.
Should be updated to read:
Emily B. Moore. ConfChem conference on interactive visualizations for chemistry teaching and learning: Accessibility for PhET Interactive Simulations—progress, challenges, and potential. (2016) _Journal of Chemical Education_. **Article ASAP**. Paper. [link: 10.1021/acs.jchemed.5b00772]

- Existing Reference: 
Moore, E. B. Designing accessible interactive chemistry simulations. (2015) Spring 2015 ConfChem: Interactive Visualizations for Chemistry Teaching and Learning.
Should be:
Emily B. Moore. Accessibility for PhET Interactive Simulations—progress, challenges, and potential. (2015) _Spring 2015 ConfChem: Interactive Visualizations for Chemistry Teaching and Learning_.

- Existing Reference: 
Emily B. Moore & Paul Grossman. ConfChem Conference on Interactive Visualizations for Chemistry Teaching and Learning: The Cutting Edge — Educational Innovation, Disability Law, and Civil Rights. _Journal of Chemical Education_. In Press.
Should be updated to read:
Emily B. Moore & Paul Grossman. ConfChem conference on interactive visualizations for chemistry teaching and learning: The cutting edge — educational innovation, disability law, and civil rights. (2016) _Journal of Chemical Education_. **Article ASAP**. Paper. [link: 10.1021/acs.jchemed.5b00795]

- Existing Reference: 
Paul Grossman, Adapted by Emily B. Moore (2015) The Cutting Edge: Educational Innovation, Disability Law, and Civil Rights. Submitted to the 2015 ConfChem Online Conference: Interactive Visualizations for Chemistry Teaching and Learning.
Should be: 
Paul Grossman, Adapted by Emily B. Moore. The cutting edge: educational innovation, disability law, and civil rights. (2015) _Spring 2015 ConfChem: Interactive Visualizations for Chemistry Teaching and Learning Chemistry_.

**Comment by ariel-phet** [comment]
@mattpen assigning to you at medium priority - seems like fairly quick changes/additions and we will be showing off our accessibility pages to many groups going forward

**Comment by mattpen** [comment]
@emily-phet - Should the text in brackets [] be copied as is, or are these instructions about the link?

**Comment by emily-phet** [comment]
@mattpen - the info in brackets are instructions regarding the link. If there aren't any bracketed instructions, then please leave the link info as it currently is on the website. Thanks!

**Comment by emily-phet** [comment]
@mattpen - one addition change:
- I noticed the link for the presentation slides for the 2016 presentation: 
February 23, 2016 – Emily B. Moore. Increasing the accessibility of PhET Simulations: Challenges, progress, and potential.

Should be the link below:
http://www.colorado.edu/csl/sites/default/files/attached-files/2016_02_23_dber-share.pdf

Right now the link just takes you to the PhET main site.

**Comment by mattpen** [comment]
[link: 10.1145/2851581.2890385]
[link: 10.1021/acs.jchemed.5b00795]
[link: 10.1021/acs.jchemed.5b00795]

Should this text just be posted as is? I'm not sure what these items mean.

**Comment by emily-phet** [comment]
@mattpen - Yes, that is confusing. For these, rather than having a link to the paper, add:
"DOI: 10.1145/2851581.2890385" to the end of each reference (with the DOI number being the corresponding 10.whatever from above). These are like pre-prints, so they don't quite have a permanent web location yet - but should be findable using the DOI number.

**Comment by mattpen** [comment]
@emily-phet - The changes have been deployed to phet-dev.  Can you please take a look and let me know if this is how they should appear?  Thanks!

**Comment by emily-phet** [comment]
@mattpen - can you send a link?

**Comment by emily-phet** [comment]
@mattpen - just checking in on this. Is it possible to send a link to the phet-dev updated site? I don't know how to access this otherwise. Thanks!

**Comment by phet-steele** [comment]
@emily-phet :smiley: http://phet-dev.colorado.edu/en/accessibility :smiley:

**Comment by emily-phet** [comment]
@phet-steele - thanks! I figured it was something simple...and it was even simpler than I thought.

**Comment by emily-phet** [comment]
Thanks @mattpen - the changes you made look great. Here are a few additions that have come up since last week: 

- The Emily Randall paper below now has a permanent link. Please replace the DOI number with the link: http://dl.acm.org/citation.cfm?id=2890385&CFID=786634772&CFTOKEN=31771063

Emily Randall. Making science simulations accessible for students with vision impairments. (2016) Proceedings of the 34th Annual ACM Conference Extended Abstracts on Human Factors in Computing Systems (pp. 122-127). ACM. Paper.

- Can you remove the listing below from the "Upcoming Presentations" section:
May 7-12, 2016 – ACM SIGCHI Conference on Human Factors in Computing Systems (CHI)
San Jose, CA.

and add this listing to the "Past Presentations - 2016" section:
May 7-12, 2016 - Emily Randall. Making science simulations accessible for students with vision impairments. ACM SIGCHI Conference on Human Factors in Computing Systems (CHI)
San Jose, CA. Paper. [link: http://dl.acm.org/citation.cfm?id=2890385&CFID=786634772&CFTOKEN=31771063]

- To the "Upcoming Presentation" listing for the STEM For All Video Showcase, can you add a link to the video so that the listing looks like: 

May 17-23, 2016 – NSF STEM For All Video Showcase
Virtual Showcase. Video. [link: http://stemforall2016.videohall.com/presentations/748]

**Comment by emily-phet** [comment]
@mattpen: can you add this one to the "Past Presentations" list?

May 4-8, 2016 - Taliesin L. Smith. Access, action, agency: Inclusive design for the non-visual use of a higher interactive simulation. OCAD University Graduate Exhibition (GradEx). Poster. [link: https://drive.google.com/file/d/0B44Uycdx6JGdNmVnbmFoOGJKTWs/view?usp=sharing]

**Comment by mattpen** [comment]
@emily-phet - All of the above changes are published at https://phet-dev.colorado.edu/en/accessibility/research

Does this look ok to publish on the main site?

Also, how often do you foresee needing to make changes to this page?  If its going to be relatively frequent we could setup an admin page so you can easily add/remove/edit entries.

**Comment by emily-phet** [comment]
@mattpen - Thanks!
I found one error that need correcting:

- The second listing under "Past Presentations" should have the title: "Access, action, agency: Inclusive design for the non-visual use of a highly interactive simulation" [right now it reads "high" instead of "higher"]

This page will need to be updating probably once every month or two. If it's pretty straightforward to set up an admin page for this that would be great. Do you have a sense of how long it would take to set up?

**Comment by mattpen** [comment]
@emily-phet - these changes were deployed to figaro today.  For now, if updates are every month or two we shouldn't need a separate admin page.  It would probably be good to add when this page is made translatable.
